add pref_episode_cache_summary

fabolhak opened the pull request:
As suggested in https://github.com/AntennaPod/AntennaPod/issues/3053#issuecomment-477224413. Unfortunately, I don't have the Android tool-chain installed and I'm not an Android developer. So it would be good if someone could test it.

ByteHamster commented:
Hi! Thanks for your PR. Unfortunately, it does not work in its current form. The preference description is overridden at run-time with the value of the preference (using Java), for example "Unlimited".

fabolhak commented:
Hello, thank you for your fast feedback. I updated the PreferenceController and removed the corresponding functions:
![Screenshot_2019-06-05_00-02-20](https://user-images.githubusercontent.com/20029691/58916878-34bf4200-8725-11e9-95cc-b95647aa617b.png)

ByteHamster commented on core/src/main/res/values-de/strings.xml:
```diff
@@ -425,6 +425,7 @@
   <string name="pref_automatic_download_on_battery_sum">Automatische Downloads auch erlauben, wenn die Batterie nicht geladen wird</string>
   <string name="pref_parallel_downloads_title">Parallele Downloads</string>
   <string name="pref_episode_cache_title">Episodenspeicher</string>
+  <string name="pref_episode_cache_summary">Gesamtzahl aller heruntergeladenen Episoden die auf dem Gerät zwischengespeichert werden. Falls diese Zahl erreicht ist, werden automatische Downloads ausgesetzt.</string>
```
Please remove the translation. Translations are only done using Transifex. Merging changes of sources and Transifex is cumbersome.

fabolhak replied:
Thank you for the comment. I removed the German translation.

ByteHamster commented:
Thanks!
